### ISSUE TITLE
bad indicating similar pictures 

### ISSUE THREAD

**Issue by Aliaksandr3by** [arsenetar/dupeguru#1091]
the program do bad indicating similar pictures .
two pictures that DupeGuru (4.3.1_64) doesn't indicate as similar.

<details>

> ![shar_2022_12_29_0561](https://user-images.githubusercontent.com/11432724/213436730-dfdf64f2-ec54-4bb6-9705-201e9506ca31.jpg)
> 
> ![shar_2023_01_01_0570](https://user-images.githubusercontent.com/11432724/213436886-47461f2c-5b98-4e1c-9b08-ac73b5f2e01e.jpg)

</details>

**Comment by arsenetar** [comment]
For the two pictures you provided you would have to have "Match pictures of different dimensions" selected.  Additionally, you may have to adjust the Filter Hardness depending on how much of a difference you want to let through, it will give an overall % match still.  For these particular ones a Filter Hardness value below 20 seems to work.

**Comment by Aliaksandr3by** [comment]
I did it, but result is absent. 

![image](https://user-images.githubusercontent.com/11432724/215006602-173e977a-95c9-4711-8c9d-a4390a914d49.png)

**Comment by Aliaksandr3by** [comment]
I understand, if I use (the filter hardness) 40% it don't recognize similar photos. but if I set 1%, it recognize.
But "match" 89%
![image](https://user-images.githubusercontent.com/11432724/215007761-d93b0fe4-5132-424c-aecc-a5ed976548ad.png)

![image](https://user-images.githubusercontent.com/11432724/215007600-5dd1426f-effb-47c7-8333-0f3cff28be69.png)

**Comment by Aliaksandr3by** [comment]
If it is work fine, the theme may be closed;

**Comment by arsenetar** [comment]
Yes the match would not be 100% it would be something smaller due to some of the visual differences.  I believe this is working as  it should, this could perhaps use some improvement on "user friendliness".

**Comment by IlluminatiWave** [comment]
Version 4.3.1. Windows 11

This is definitely not working (it even affects files).

Context
I am comparing folders with jpg's, png's with others with jxl's. I thought it was just ignoring the JXL's (since it finds no difference with these formats, even if I copy and paste the same jxl it doesn't find it in image mode nor in standard / file mode).

I created a test folder with the jxl images and it finds no similarities (I thought there was already support for jxl in Windows). But I did the same test with the cjxl.exe executable and it didn't succeed either. I had to lower the hardness to the minimum so it could find them (both with 67% similarity). and even then it fails (it finds the similarity between different files, not between themselves).

Log:

2023-04-05 19:08:48,793 - DEBUG - Debug mode enabled
2023-04-05 19:09:02,610 - DEBUG - Collected 4 files in folder C:\Users\IW\Downloads\moviendo\Nueva carpeta
2023-04-05 19:09:02,611 - INFO - Scanning 4 files
2023-04-05 19:09:02,611 - INFO - Getting matches. Scan type: 0
2023-04-05 19:09:02,611 - DEBUG - Reading metadata of C:\Users\IW\Downloads\moviendo\Nueva carpeta\1 - copia.webp
2023-04-05 19:09:02,611 - DEBUG - Reading metadata of C:\Users\IW\Downloads\moviendo\Nueva carpeta\1.webp
2023-04-05 19:09:02,612 - DEBUG - Reading metadata of C:\Users\IW\Downloads\moviendo\Nueva carpeta\cjxl - copia.exe
2023-04-05 19:09:02,612 - DEBUG - Reading metadata of C:\Users\IW\Downloads\moviendo\Nueva carpeta\cjxl.exe
2023-04-05 19:09:02,612 - INFO - Found 3 matches
2023-04-05 19:09:02,613 - INFO - Grouping matches
2023-04-05 19:09:02,613 - INFO - Created 1 groups

**Comment by qwewer0** [comment]
I tested dupeGuru with a large image set in Picture mode.
Below are the numbers of duplicate pairs found at each Filter Hardness (columns are the % similar pairs found).

| Filter Hardness | 100% | 99% | 98% | 97% | 96% | 95% | <95 |
|-----------------|------|-----|-----|-----|-----|-----|-----|
| 100             | 16   | -   | -   | -   | -   | -   | -   |
| 99              | 16   | 7   | -   | -   | -   | -   | -   |
| 98              | 16   | 12  | 0   | -   | -   | -   | -   |
| 97              | 16   | 15  | 2   | 0   | -   | -   | -   |
| 96              | 16   | 16  | 4   | 3   | 0   | -   | -   |
| 95              | 16   | 16  | 4   | 4   | 1   | 0   | -   |
| 75              | 16   | 10  | 3   | 6   | 3   | 4   | 39  |
| 50              | 16   | 10  | 3   | 6   | 3   | 3   | 95  |
| 25              | 16   | 10  | 3   | 6   | 3   | 4   | 218 |
| 1               | 16   | 8   | 3   | 5   | 3   | 4   | 591 |

As shown, the number of pairs found is sometimes less and sometimes more when lowering the Filter Hardness
(for example, 99 finds fewer 99%-similar pairs than 98).

It's the same with a built dupeGuru from source (latest master)
